code/admin/admin_verbs_debug.dm

```diff
@@ -108,16 +117,10 @@
 				src << output("[d.type] ([d])", "[window_ref].varsgrid:2,[i++]")
 			else if(k == "appearance")
 				src << output("/appearance", "[window_ref].varsgrid:2,[i++]")
-			else if(k == "flags" && istype(O, /atom))
-				var/flag_list = "bitflags: [value]"
-				var/bit = 1
-				for(var/j = 1 to 16)
-					if(value & bit)
-						flag_list = "[flag_list]\n   [atomflag2name(bit)]"
-					bit = bit<<1
-				src << output(flag_list, "[window_ref].varsgrid:2,[i++]")
-			else if((k == "sight" && ismob(O)) || (istype(O, /atom) && k in list("appearance_flags", "blend_mode")))
-				src << output(FlagsToBits(value), "[window_ref].varsgrid:2,[i++]")
+			else if(k == "blend_mode")
+				src << output("mode: [__blend_mode_flags["[value]"]]", "[window_ref].varsgrid:2,[i++]")
+			else if(isnum(value) && k in list("flags", "sight", "appearance_flags"))
```

Comment:
BYOND's in-operator is a bit wonky in terms of operator precedence.
To be on the same side you always want to surround an in-operator with brackets, i.e. 
```else if(isnum(value) && (k in list("flags", "sight", "appearance_flags")))```